### ISSUE TITLE
Add tag to hide editor entirely (for plugins)

### ISSUE THREAD

**Issue by dreamflasher** [benji300/joplin-persistent-layout#14]
Plugins such as Kanban and JSheets don't need the editor view, in fact it's stealing valuable screen space. 
It would be great if the presistant layout plugin would have an additional tag `layout:hide`, which hides the editor/richtext view.

**Comment by MattDemers** [comment]
Agreed; I found this plugin mostly to have a "clips" notebook that didn't need to be edited at all, and to default to the "viewer" layout.
